specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -196,8 +198,9 @@ Process A (Registration of the Push Trigger only):
 ```cpp
 int main()
 {
+    PushNotificationRegistrationOptions pushOptions = PushNotificationManager::GetPushActivatorKind(false);
```

Comment:
GetPushActivatorOptions(false);??

Reply:
This API call will be the equivalent of IsSupported, which will return flags for the developer to call into RegisterActivator. Packaged apps should set this isInProcActivation parameter to true if the calling process will handle background activation.